README.md

```diff
@@ -15,8 +15,16 @@ We encourage you to participate in this open source project. We love Pull Reques
 
 High-level components for building browser(-like) apps.
 
+* **Engine** - An abstract layer on hiding the actual browser engine implementation.
```

Comment:
nit: remove 'on' or for hiding? to hide?

Reply:
Oops. Yeah, I rewrote that sentence a couple of times 😅